src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/test/WattTimeDataSourceTests.cs

```diff
@@ -308,15 +252,70 @@ public void GetCarbonIntensity_ConvertsMoerToGramsPerKwh(float lbsPerMwhEmission
         Assert.That(result * GRAMS_TO_POUNDS / KWH_TO_MWH, Is.EqualTo(lbsPerMwhEmissions).Within(FLOATING_POINT_TOLERANCE));
     }
 
-    private void SetupBalancingAuthority(BalancingAuthority balancingAuthority, Location location)
+    /// <summary>
```

Comment:
I would make a note that the actual changes in this file (aside from refactors) actually start here.

Reply:
Added a link in the PR description and will leave this comment open until we're ready to merge👍